New version: MCHammer v0.2.0

JuliaRegistrator opened the pull request:
- Registering package: MCHammer
- Repository: https://github.com/etorkia/MCHammer.jl
- Created by: @etorkia
- Version: v0.2.0
- Commit: 0673837f2af5315bf95ca4fbe0d693b1d4d27c89
- Reviewed by: @etorkia
- Reference: https://github.com/etorkia/MCHammer.jl/issues/2#issuecomment-2781061964
- Description: Monte Carlo Simulation and Business Analysis tools for Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Changelog

- We update compats
- Added new functions
- Detailed the documentation

[noblock]
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598d744f4e177f5eafe8672c75586fd6677ade9b1ca48ba1541d51545d6de94044906d17de2bbc19de283a5ff74c6bf607f90066ee2afe4dd582dd1b6ab7b693e0e4162739fd0c503e409b11918087dca610bfb5c937f8cc053284bea4ec3eda966965b9c69a4dafd62e43071ec8e845eb84dfe5e5dec68fb861106bd320fa2c06e -->

etorkia commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- We update compats
- Added new functions
- detailed the documentation
- ...

etorkia commented:
@JuliaRegistrator register

Release notes:
Breaking changes

    We update compats
    Added new functions
    detailed the documentation
    ...

etorkia commented:
@JuliaRegistrator register

Release notes:
Breaking changes

We update compats
Added new functions
detailed the documentation
...


[noblock]

etorkia commented:
@JuliaRegistrator register

# Breaking changes
No breaking changes
[noblock]

etorkia commented:
@JuliaRegistrator register

Release notes:
## Changelog

We update compats
Added new functions
detailed the documentation
...

etorkia commented:
@JuliaRegistrator register

Release notes:
## Changelog

We update compats
Added new functions
detailed the documentation
...

etorkia commented:
@JuliaRegistrator register

Release notes:
## Changelog

We update compats
Added new functions
detailed the documentation
...

JuliaRegistrator committed:
New version: MCHammer v0.2.0

UUID: 58716550-704d-11e9-05c2-8f708e47025f
Repo: https://github.com/etorkia/MCHammer.jl.git
Tree: b04590d9d1e54c167fbc98a49fd0dda02fc3e354

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

etorkia commented:
@JuliaRegistrator register

Release notes:
## Changelog

We update compats
Added new functions
detailed the documentation
[noblock]
